R/standalone-lifecycle.R

```diff
@@ -235,12 +239,10 @@ with_lifecycle_errors <- function(expr) {
 
   if (!rlang::is_string(opt, c("quiet", "default", "warning", "error"))) {
     options(lifecycle_verbosity = "default")
-    rlang::warn(glue::glue(
-      "
-      The `lifecycle_verbosity` option must be set to one of:
-      \"quiet\", \"default\", \"warning\", or \"error\".
-      Resetting to \"default\".
-      "
+    rlang::warn(paste(
```

Comment:
Just noting that this is changing the formatting (I think I used glue for newlines). That doesn't really matter though.